src/code/z_player_lib.c

```diff
@@ -2229,38 +2966,397 @@ struct_80128388_arg1 D_801C0C54[] = {
     { 30.0f, 0x0000, 0x0000, { 0.0f, 0.0f, 0.0f }, 20.0f, 0x1F40, 0x2EE0 },
 };
 
-Color_RGB8 D_801C0CA8[] = {
-    { 255, 255, 255 }, { 80, 80, 255 },   { 136, 192, 255 }, { 136, 192, 255 }, { 184, 232, 232 }, { 248, 200, 0 },
-    { 255, 180, 0 },   { 0, 128, 0 },     { 252, 238, 0 },   { 131, 0, 174 },   { 64, 64, 32 },    { 0, 0, 255 },
-    { 255, 0, 255 },   { 255, 0, 255 },   { 255, 0, 0 },     { 0, 0, 255 },     { 0, 200, 0 },     { 255, 255, 255 },
-    { 255, 255, 255 }, { 255, 255, 255 }, { 80, 80, 255 },
+Color_RGB8 D_801C0CA8[PLAYER_BOTTLE_MAX] = {
+    { 255, 255, 255 }, // PLAYER_BOTTLE_EMPTY            // PLAYER_IA_BOTTLE
+    { 80, 80, 255 },   // PLAYER_BOTTLE_FISH             // PLAYER_IA_BOTTLE_FISH
+    { 136, 192, 255 }, // PLAYER_BOTTLE_SPRING_WATER     // PLAYER_IA_BOTTLE_SPRING_WATER
```

Comment:
I don't think commenting on the `IA` enums is necessary, the `PLAYER_BOTTLE_` enums should be enough